README.md

```diff
@@ -15,3 +15,9 @@ the function and run automatically when you commit through a GitHub action.
 ## Runtime Analysis
 
 What is the worst-case big $\Theta$ time complexity of your algorithm?
+
+I had help creating the test code from online and other github assignments. I also had help from my brother with the logic of isomorphism.
+
+The worst case big Theta time complexity of my algorithm would be big theta (n! n^2 log n). The reason it would be this bad is if for n nodes the max number of permutations must be checked, and if the adjacensy list are dense maxing the time cost of sorting neighbors. I used brute force method for generating all permustations for node mappings, and each permutation is checked which adds the process of sorting and comparing.
```

Comment:
Where does the $n^2 \log n$ come from?

Reply:
each node is mapped and sorted giving n log n and then the other instance of n is that inside that loop there are n nodes so n log n happens n times making n^2 log n.